### DISCUSSION TITLE
Remove the warning message due to using the deprecated package

### DISCUSSION THREAD

**Discussion by git-hulk** [Kong/kong#12859] [Help]
It will throw the warning message while running the Kong
due to using the deprecated packages `pl.text`:

```
[warn] the contents of module 'pl.text' has moved into 'pl.stringx'
```

Does it make sense to resolve this warning message by replacing
with the new package? **If yes, I will happily submit a PR to do this.**

**Comment by chronolaw** [comment]
Great! Please create a PR to fix it, thank you very much.

**Reply by git-hulk** [reply]
Thanks for your quick response, I will do it soon.

**Reply by git-hulk** [reply]
I'm sorry that this issue has already been resolved.
